docs/examples/http_exceptions.py

```diff
@@ -0,0 +1,49 @@
+import json
+
+from nameko.web.handlers import HttpRequestHandler
+from werkzeug.wrappers import Response
+
+
+class ApplicationError(Exception):
+    pass
+
+
+class HttpError(Exception):
+
+    def __init__(self, payload, status_code=None):
```

Comment:
Once `response_from_exception` is overridden, client's only option to raise different http errors would be to carry that information in exception. To be more explicit other helper exceptions that inherit from `HttpError` could be created by client like `InvalidAgruments (422)`, `EntityNotFound (404)` or `BadRequest (400)` etc...